CSS/works-styles.css

```diff
@@ -0,0 +1,136 @@
+.works {
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  height: 500vh;
+  padding: 0;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.work-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  width: 327px;
+  height: fit-content;
+  border-radius: 16px;
+  background-color: #fff;
+}
```

Comment:
- [ ] Kindly reduce the space between the home page and the cards so that the design should match to the Figma design 
 ## your design
![image](https://user-images.githubusercontent.com/95297251/185513073-0cf98763-ae57-497d-8d40-091376771f2c.png)
## Figma design
![image](https://user-images.githubusercontent.com/95297251/185513588-76902edc-6f83-4859-b7b9-654f5d6f822d.png)

Reply:
hi there, thanks for the reviews, but doesn't it move according to the height of the screen? and if it does or not should I reduce them anyways?